coursestyle.css

```diff
@@ -0,0 +1,7 @@
+.body {
+	font-family: 'Montserrat', sans-serif;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/shiny-tribble/pullRequest?prid=5485358)